### PR TITLE
installed pgloader

### PR DESCRIPTION
**Pull request by kzidane**

### CODE REVIEW

**Comment by dmalan** [comment]
Lemme play with this a bit more before we merge.

File: update50 [hunk 1]
```diff
@@ -49,6 +49,15 @@ if [ ! -f /etc/apt/sources.list.d/github_git-lfs.list ]; then
         -o Dir::Etc::sourceparts="-"
 fi
 
+# install pgloader apt repo
+# https://wiki.postgresql.org/wiki/Apt
+if [ ! -f /etc/apt/sources.list.d/pgdg.list ]; then
+    echo "deb http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -cs)-pgdg main" | sudo tee /etc/apt/sources.list.d/pgdg.list
+    curl -sSL https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+    sudo apt-get update -o Dir::Etc::sourcelist="sources.list.d/pgdg.list" \
+        -o Dir::Etc::sourceparts="-"
```
Comment (dmalan) [inline update50]: Any reason this is on two lines but earlier lines are longer anyway?

**Comment by dmalan** [comment]
@brianyu28 mind testing these lines on your own workspace too, followed by an `apt-get update && apt-get install -y pgloader`, to ensure you end up with version 3.x instead of 2.x?

https://github.com/cs50/ide50/pull/213/files#diff-aca93b2492941d6c349c60fb3e4f8fcbR52

**Comment by brianyu28** [comment]
@dmalan needed to use `sudo apt-get update && sudo apt-get install -y pgloader`, but otherwise seems to have worked!

**Comment by kzidane** [comment]
Should be deployed now!

**Comment by dmalan** [comment]
Thanks!

**Pull request merged**